web_clickjack_protection/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Just install
```

Comment:
OCA policy is to remove this if there is nothing sensible to do, so let's just remove it.

Reply:
Fixed